core/src/solution_submission.rs

```diff
@@ -9,10 +9,9 @@ use ethcontract::errors::{ExecutionError, MethodError};
 use ethcontract::web3::types::TransactionReceipt;
 use ethcontract::U256;
 use futures::future::{BoxFuture, FutureExt as _};
-use log::info;
```

Comment:
This file had mixed uses of logging macros `log::info!` vs `info!`. I prefer having the module name to see where this is from especially for `log::error!`.

Reply:
I also prefer this for the log macros. I think a lot of the `use log::{info, error}` come from the migration from edition 2015 to 2018, where the `#[macro_use] extern crate log;` was removed.